lib/utils/getServerlessConfigFile.test.js

```diff
@@ -32,7 +32,7 @@ describe('#getServerlessConfigFile()', () => {
     const serverlessFilePath = path.join(tmpDirPath, 'serverless.yml');
 
     writeFileSync(serverlessFilePath, 'service: my-yml-service');
-    return expect(getServerlessConfigFile(tmpDirPath)).to.be.fulfilled.then((result) => {
+    return expect(getServerlessConfigFile(undefined, tmpDirPath)).to.be.fulfilled.then((result) => {
```

Comment:
Nit: Literally I think it's cleaner to use `null` for no value (intentional) input instead of `undefined`

Reply:
i used `undefined` because IIRC thats what `...options.config` will be if the user doesn't specify `--config`, so this is closer to how `getServerlessConfigFile` will actually be called.

Reply:
Technically there's no (and there should be no) difference in how `null` and `undefined` are handled. Fact that this function may receive `null` in some cases and `undefined` in others is perfectly fine.

It's just that semantically when inputting a _no value_ explicitly `null` seems a more clean choice. It's intentional _no value_, while `undefined` feels more accidental, e.g. it's the outcome of accessing a potentially defined property.

I personally try to not use `undefined` in literal form, also in older versions of a language it had implications as it wasn't a keyword, just a global variable easy to override.

So having `null` here, just feels better imo (but it was just a comment :)

Reply:
I once read that `undefined` is what's computed / "picked" by Node.js / JavaScript and "humans" should always use `null`, never `undefined` (if that writeup makes sense).

That's the rule I'm always trying to follow following. Not sure if it's legit.